src/main/resources/alma/common/contribution.xml

```diff
@@ -13,10 +17,10 @@
         <call-macro name="gndIdentifier" field="100" />
         <call-macro name="gndIdentifierId" field="100" />
         <!--
-      <call-macro name="personTitle"      field="100" />
-      <call-macro name="personBio"        field="100" />
-      <call-macro name="personRole"       field="100" />
-      <call-macro name="gndIdentifier"    field="100" />
+      <call-macro name="personTitle" field="100" />
+      <call-macro name="personBio" field="100" />
+      <call-macro name="personRole" field="100" />
+      <call-macro name="gndIdentifier" field="100" />
```

Comment:
at least this comment about macro calling is redundant (see line 17). I would suggest to remove all this comments here)

Reply:
Ok this seems to be a formatting highlighting. Perhapse I should tidy up the outcommented macros in a new branch since they are not connected to the responsibilityStatement

Reply:
+1. So you will remove the formatting highlighting here as it also has nothing to do with the responsibilityStatement?